integration_test/common/testdata/integration-metadata_onetrue_expected-metric/input.yaml

```diff
@@ -0,0 +1,31 @@
+app_url: "https://example.google.com/"
+short_name: OpsAgent
+long_name: Cloud OpsAgent
+description: |-
+  Example description
+minimum_supported_agent_version:
+  metrics: 1.0.0
+  logging: 1.0.0
+supported_app_version: ["Example0"]
+expected_metrics:
```

Comment:
This test file is testing that none of the expected metrics has the representative field. There shall be another test that is on when there are more than one expected metrics that have representative field. Thus it's matching `there must be exactly one metric with representative: true`, cite from: https://github.com/GoogleCloudPlatform/ops-agent/blob/2e5fe210186b12e5c175fc9344d26e7bac151759/integration_test/common/common.go#L111